src/applications/mhv/shared/util/helpers.js

```diff
@@ -44,3 +44,10 @@ export const generatePdfScaffold = (user, title, subject, preface) => {
     preface,
   };
 };
+
+/**
+ * @param {String} newTitle title of the page, displayed on the browsers title bar, should not be the same across different pages
+ */
```

Comment:
description may be simplified, as the latter part sounds more of an acceptance criteria rather then a general description of a function 
```suggestion
/**
 * Sets a page title of the document
 * @param {String} newTitle title of the page, displayed on the browsers title bar
 */
```